src/ansys/simai/core/data/models.py

```diff
@@ -120,3 +233,24 @@ def build(
                 dismiss_data_with_volume_overflow,
             )
         )
+
+    def doa_axis_definition(
+        self, position: str, value: float, length: PositiveFloat
+    ) -> DomainAxisDefinition:
+        """Sets an axis in the Domain of Analysis.
+
+        Returns:
+            a :class:`DomainAxisDefinition <DomainAxisDefinition>` object.
+
+        Raises:
+            `ansys.simai.core.errors.InvalidArguments`: when the arguments are not compatible with the definition of an axis of the Domain of Analysis.
```

Comment:
`ansys.simai.core.errors.InvalidArguments`: When the arguments are not compatible with the definition of an axis within the Domain of Analysis.